core-api/src/controllers/checkoutSlot.controller.ts

```diff
@@ -0,0 +1,51 @@
+import { Request, Response } from "express";
+import { HttpStatus } from "../utils/httpStatus";
+import { ServerError, ErrorCode } from "../utils/serverError";
+import messages from "../utils/messages";
```

Comment:
In other files, "messages" is "Messages", it would be nice to have a standard

Reply:
done